index.html

```diff
@@ -0,0 +1,417 @@
+        <ul class="features__list">
+          <li class="features__item">
+            <div class="features__card">
+              <img src="./assets/shield.svg" alt="shield">
+              <h4 class="features__card-heading">
+                High security to protect from piracy
+              </h4>
+              <p class="features__card-text">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+                diam nonumy.
+              </p>
+            </div>
+          </li>
+          <li class="features__item">
+            <div class="features__card">
+              <img src="./assets/thumb.svg" alt="shield">
+              <h4 class="features__card-heading">Premium quality performance</h4>
+              <p class="features__card-text">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+                diam nonumy.
+              </p>
+            </div>
+          </li>
+          <li class="features__item">
+            <div class="features__card">
+              <img src="./assets/earphone.svg" alt="shield">
+              <h4 class="features__card-heading">
+                Full time customer support - 24/7
+              </h4>
+              <p class="features__card-text">
+                Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+                diam nonumy.
+              </p>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="about-us">
+      <div class="about-us__header">
+        <h4 class="about-us__label">Why should you work with us?</h4>
+        <h2 class="about-us__heading">
+          To upscale your business to the next level
+        </h2>
+        <p class="about-us__text">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+          erat, sed diam voluptua invidunt ut labore.
+        </p>
+      </div>
+      <div class="about-us__body">
+        <ul class="about-us__list">
+          <li class="about-us__item">
+            <img src="./assets/arrow.svg" alt="arrow">
+            <p class="about-us__item-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </li>
+          <li class="about-us__item">
+            <img src="./assets/arrow.svg" alt="arrow">
+            <p class="about-us__item-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </li>
+          <li class="about-us__item">
+            <img src="./assets/arrow.svg" alt="arrow">
+            <p class="about-us__item-text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod.
+            </p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="cta">
```

Comment:
Would you improve this block `cta`
we should use meaningful names

Reply:
that was the name, they gave in the Figma file, which Maurice advice to use

Reply:
Awesome, Figma can be helpful for that but I think Figma is going to give you an inspiration only It does not have to define out block